python/cudf/cudf/core/series.py

```diff
@@ -1841,6 +1841,60 @@ def round(self, decimals=0):
             dtype=self.dtype,
         )
 
+    def kurtosis(self, axis=None, skipna=True, numeric_only=True):
```

Comment:
Add level kwargs and throw exception if non-default for now for API compatibility